controllers/channel.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const winston = require('winston');
+const util = require(__dirname + '/../helpers/util');
+const Channel = require(__dirname + '/../models/channel');
+
+
+
+exports.get_channels_by_user = (req, res, next) => {
+    const data = util.get_data({
```

Comment:
Data can possibly return a new(Error), so you might do something like data.attribute = data.attribute || arbitrary_value